numpy/core/fromnumeric.py

```diff
@@ -3125,10 +3125,35 @@ def around(a, decimals=0, out=None):
     -----
     For values exactly halfway between rounded decimal values, NumPy
     rounds to the nearest even value. Thus 1.5 and 2.5 round to 2.0,
-    -0.5 and 0.5 round to 0.0, etc. Results may also be surprising due
-    to the inexact representation of decimal fractions in the IEEE
-    floating point standard [1]_ and errors introduced when scaling
-    by powers of ten.
+    -0.5 and 0.5 round to 0.0, etc.
+
+    `np.around` uses a fast but sometimes inexact algorithm to round
+    floating-point datatypes. For positive `decimals` it is equivalent to
+    `np.true_divide(np.rint(a * 10**decimals), 10**decimals)`, which is inexact
+    for large floating-point values or large values of `decimal` due the
+    inexact representation of decimal fractions in the IEEE floating point
+    standard [1]_ and errors introduced when scaling by powers of ten. For
+    instance, note the extra "1" in the following:
```

Comment:
Does this need "::'? I'm a bit concerned that some of these might make it as doctests.  Or is that intended? Most doctests are not indented.

Reply:
It seems to work, see the [rendered doc](https://9043-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.around.html#numpy.around)

Reply:
nit: decimals is a parameter, so it only really needs single backticks not double. It is find as code as well, so I am not sure it has to be changed.